File: Selenium/Activities/Activity9.java
package activity;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity9 {

	public static void main(String[] args) {
 
	        WebDriver driver = new FirefoxDriver();
            driver.get("https://training-support.net/webelements/dynamic-content");
	        driver.manage().window().maximize();
            System.out.println("Page Title: " + driver.getTitle());

            WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
	        WebElement clickButton = driver.findElement(By.xpath("//button[text()='Click me!']"));
	        clickButton.click();

            wait.until(ExpectedConditions.textToBePresentInElementLocated(By.tagName("body"),"release"));
            String pageText = driver.findElement(By.tagName("body")).getText();
            System.out.println("Page Text contains 'release':");
            System.out.println(pageText);

            driver.quit();
	        
	    }
	}

File: Selenium/Activities/activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC


driver = webdriver.Firefox()

try:
    
    driver.get("https://training-support.net/webelements/dynamic-content")
    driver.maximize_window()

    
    print("Page Title:", driver.title)

    
    wait = WebDriverWait(driver, 10)

    
    click_button = driver.find_element(By.XPATH, "//button[text()='Click me!']")
    click_button.click()

    
    wait.until(
        EC.text_to_be_present_in_element(
            (By.TAG_NAME, "body"),
            "release"
        )
    )

    
    page_text = driver.find_element(By.TAG_NAME, "body").text

    print("Page Text contains 'release':")
    print(page_text)

finally:
    
    driver.quit()